Rudder-Braze/Classes/RudderBrazeIntegration.m

```diff
@@ -18,7 +18,17 @@ - (instancetype)initWithConfig:(NSDictionary *)config withAnalytics:(nonnull RSC
         self.config = config;
         self.client = client;
         self.supportDedup = [[config objectForKey:@"supportDedup"] boolValue] ? YES : NO;
-        NSString *apiToken = [config objectForKey:@"appKey"];
+
+        BOOL usePlatformSpecificKeys = [[config objectForKey:@"usePlatformSpecificKeys"] boolValue];
+        NSString *apiToken = @"";
+        // Prefer platform-specific key if present and not empty
+        if (usePlatformSpecificKeys && [config objectForKey:@"iOSAppKey"]) {
+            apiToken = [config objectForKey:@"iOSAppKey"];
```

Comment:
The config parameter is called "key" whereas the variable is "token". Let's be consistent.

Reply:
I’ve followed the existing naming convention to avoid triggering additional changes elsewhere, which could complicate the review and shift the focus of this PR. I believe, ideally, such updates should be handled in a separate PR for easy review.

I'll rename the `apiToken` to `apiKey`.

Reply:
I agree. Will try to follow this approach going forward.